process-deploy/src/main/java/com/wipro/ats/bdre/pdeploy/ProcessDeploy.java

```diff
@@ -61,7 +61,7 @@ public void run() {
         }
         String[] params = {"--deployment-id", pdq.getDeploymentId().toString()};
         //using Apache Commons Exec library
-        int iExitValue;
+
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "iExitValue" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)